src/main/java/com/contrastsecurity/ui/com/contrastsecurity/ui/toolwindow/ContrastToolWindowFactory.java

```diff
@@ -994,6 +993,58 @@ private void populateVulnerabilityDetailsRecommendation(RecommendationResource r
         }
     }
 
+    public void parseHtmlLinksAddToPanel(JTextPane text, JPanel panel) {
```

Comment:
should this be in ContrastUtil? seems similar to parseMustache

Reply:
if this is specific to the toolwindowfactory we should make it private.

Reply:
Because it builds onto the panel and adds a mouse listener, I put it in the ToolWindowFactory. Might deserve a rename though.

Reply:
does it need to be public?

Reply:
probably not, I can change it to private.